catalog_integrations_api_tools/sftp.py

```diff
@@ -30,7 +52,26 @@ def get_all_files(
         Returns:
             str: Name of directory the files were downloaded to.
         """
-        pass
+        products_pattern_compiled = re.compile(
+            f"{file_prefix}_products_" + r"\d{14}\.csv(\.gz)?"
+        )
+        stockprices_pattern_compiled = re.compile(
+            f"{file_prefix}_stock-prices_" + r"\d{14}\.csv(\.gz)?"
+        )
```

Comment:
You don't actually need to compile the regular expressions.
```suggestion
        products_pattern_compiled = f"{file_prefix}_products_" + r"\d{14}\.csv(\.gz)?"
        stockprices_pattern_compiled = f"{file_prefix}_stock-prices_" + r"\d{14}\.csv(\.gz)?"
```